packages/pigeon/pigeons/core_tests.dart

```diff
@@ -39,7 +39,7 @@ class AllTypes {
   Int64List a8ByteArray;
   Float64List aFloatArray;
   // ignore: always_specify_types, strict_raw_type
-  List aList;
+  List list;
```

Comment:
I think it's worth commenting on why this name doesn't match all the other ones, so someone doesn't think it's a mistake a change it back later.